### PR TITLE
Suggestions for deprecating older WebP versions

### PR DESCRIPTION
**Pull request by radarhere**
Suggestions for https://github.com/python-pillow/Pillow/pull/8213

### CODE REVIEW

File: src/PIL/features.py [hunk 1]
```diff
@@ -180,7 +180,14 @@ def get_supported_features() -> list[str]:
     """
     :returns: A list of all supported features.
     """
-    return [f for f in features if check_feature(f)]
+    supported_features = []
+    for f, (_, flag, _) in features.items():
+        if flag is True:
+            if check_module("webp"):
```
Comment (homm) [inline src/PIL/features.py]: What is purpose of this change? check_feature Will also check module

Comment (radarhere) [reply src/PIL/features.py]: It prevents `get_supported_features()` from raising a deprecation warning.

Comment (homm) [reply src/PIL/features.py]: I don't like `check_module("webp")` hardcoding here. We could either use path from Feature to check the module, or still use `check_feature` function and suppress depiction warning

Comment (radarhere) [reply src/PIL/features.py]: Ok, I've pushed a dynamic version, see what you think.

**Pull request merged**